ergotree-ir/src/sigma_protocol/dlog_group.rs

```diff
@@ -105,6 +107,63 @@ pub fn random_scalar_in_group_range() -> Scalar {
     Scalar::generate_vartime(&mut OsRng)
 }
 
+/// Attempts to create BigInt from Scalar
+pub fn scalar_to_bigint(s: Scalar) -> BigInt {
```

Comment:
A good final touch would be a roundtrip test. No need for PBT, simple one-off `dlog_group::random_scalar_in_group_range()` value converted to BigInt and back.